### PR TITLE
Create Ruotong's Ch5 Exercise Post

### PR DESCRIPTION
**Pull request by izayak**

### CODE REVIEW

File: _posts/izayak/2016-02-03-ch5-exercise.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+author: izayak
+title: "Ruotong's Chapter 5 Loops Exercise"
+layout: post
+---
+
+<iframe src="https://trinket.io/embed/python/cb8bf3c733" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```
Comment (eah13) [inline _posts/izayak/2016-02-03-ch5-exercise.md:7]: Mostly good.  When I entered 5, 10, 0 in exercise 2, though, the average was shown as 10, not 5.  Can you figure out why?

File: _posts/izayak/2016-02-03-ch5-exercise.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Thoughts  
+I just use the same 'count', 'total' and 'num' variable names in the two exercises. So if in the first one the user hits enter button
+before inputing 'done', the nan of result of average will pass to the the next one. But if the user inputs done directly without hitting
```
Comment (eah13) [inline _posts/izayak/2016-02-03-ch5-exercise.md:11]: `nan` is definitely something to contend with.  We'll discuss it in class.

**Comment by eah13** [comment]
:mag_right:

**Comment by izayak** [comment]
I should put a start value of total. @eah13 
And about catching the no input problem, I forgot the discussion in class. Could you say how to do that again? I think 
 if not input 
should work but it does not...

**Comment by eah13** [comment]
There are several ways to deal with it.  Can you detect when the user has input nothing and change the behavior of your program in that case?  We have the tools to do this with what we've learned so far.

**Comment by izayak** [comment]
@eah13 Thanks! I got it!

**Pull request merged**